src/store/model-store.js

```diff
@@ -31,4 +34,8 @@ store.registerHandler('NO_MODEL_FOUND', () => {
     store.emitChange();
 });
 
+store.registerHandler('MODELS_RECEIVED', data => {
```

Comment:
Models received is a weird name for an action that is supposed to get the models.  Something like RETRIEVE_MODELS or FIND_MODELS seems more appropriate.